fail2ban/client/configreader.py

```diff
@@ -285,8 +285,10 @@ def getOptions(self, pOpts):
 		
 		if self.has_section("Init"):
 			for opt in self.options("Init"):
+				v = self.get("Init", opt)
```

Comment:
OK, I will just do it myself after the merge if that is so much effort

On November 2, 2015 5:32:07 PM EST, "Serg G. Brester" <notifications@github.com> wrote:
>> @@ -285,8 +285,10 @@ def getOptions(self, pOpts):
>>  		
>>  		if self.has_section("Init"):
>>  			for opt in self.options("Init"):
>> +				v = self.get("Init", opt)
>
>`v` will be use in 2 lines below. Please let such easy code fixing not
>as a hand brake for such PR's again. I mean you have apparently no
>problem with the `opt` above...
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/fail2ban/fail2ban/pull/1242/files#r43693041